applications/serial_lte_modem/doc/Generic_AT_commands.rst

```diff
@@ -116,32 +116,32 @@ The test command is not supported.
 Power saving #XSLEEP
 ====================
 
-The ``#XSLEEP`` command makes the nRF9160 development kit enter idle or sleep mode.
+The ``#XSLEEP`` command makes the nRF9160 System in Package (SiP) enter idle or sleep mode.
```

Comment:
Here the modes are sleep and idle, but in the syntax section they are Sleep and Idle. Which is the correct version?

Also, for the shutdown, it uses a third version: SYSTEM OFF mode. Is there a reason it is capitalized differently?

Reply:
Sleep and Idle, modes defined by application
System Off, nRF9160 power management mode (see PS)